src/test/java/de/tum/cit/aet/artemis/iris/PyrisEventSystemIntegrationTest.java

```diff
@@ -264,7 +244,7 @@ void testShouldNotFireProgressStalledEventWithEventDisabled() {
         createSubmissionWithScore(studentParticipation, 40);
         createSubmissionWithScore(studentParticipation, 40);
         var result = createSubmissionWithScore(studentParticipation, 40);
-        assertThatExceptionOfType(AccessForbiddenAlertException.class).isThrownBy(() -> pyrisEventService.trigger(new NewResultEvent(result)));
+        verify(pyrisEventService, times(0)).trigger(new NewResultEvent(result));
```

Comment:
You can use `Mockito.never()` instead of `times(0)`
```suggestion
        verify(pyrisEventService, never()).trigger(new NewResultEvent(result));
```